common/data_refinery_common/migrations/0002_auto_20180409_1908.py

```diff
@@ -0,0 +1,95 @@
+# Generated by Django 2.0.2 on 2018-04-09 19:08
```

Comment:
Rolling it into initial will break migrations for development. If we want to avoid going past the initial one then we should just not source control the migration files so that `./update_models.sh` just does the right thing for each dev.